src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -47,7 +47,7 @@ class ConstraintViolationBuilder implements ConstraintViolationBuilderInterface
     /**
      * @param TranslatorInterface $translator
      */
-    public function __construct(ConstraintViolationList $violations, Constraint $constraint, $message, array $parameters, $root, $propertyPath, $invalidValue, $translator, $translationDomain = null)
+    public function __construct(ConstraintViolationList $violations, Constraint $constraint, string $message, array $parameters, $root, string $propertyPath, $invalidValue, $translator, string $translationDomain = null)
```

Comment:
this breaks the 3.4 CI, so let's allow null

Reply:
see https://travis-ci.org/symfony/symfony/jobs/546867641

Reply:
The job you linked succeded?

Reply:
Yes, I relaunched it after #32074 and it passed.